Etabs_Adapter/CRUD/Create/Bar.cs

```diff
@@ -153,11 +154,15 @@ private bool SetObject(Bar bhBar)
                 offset2[2] = offset.End.Y;
             }
 
+
+#if Debug21 == false && Release21 == false
+
             if (m_model.FrameObj.SetInsertionPoint(name, (int)bhBar.InsertionPoint(), false, bhBar.ModifyStiffnessInsertionPoint(), ref offset1, ref offset2) != 0)
             {
                 CreatePropertyWarning("Insertion point and perpendicular offset", "Bar", name);
                 ret++;
             }
+#endif
```

Comment:
Does this actually work in practice? You can simplify the if statement, but just checking `#if Debug21' - VS has quite a good UI feature that highlights the code that is active dependent on the build configuration.